streaming/jms-connector/src/main/java/JmsQueueSample.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.core.JobStatus;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.Sources;
+import org.apache.activemq.ActiveMQConnectionFactory;
+
+import javax.jms.TextMessage;
+
+import static com.hazelcast.jet.impl.util.Util.uncheckCall;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * A pipeline which streams messages from a JMS queue, filters them according
+ * to the priority and writes a new message with modified properties to another
+ * JMS queue.
+ */
+public class JmsQueueSample {
+
+    private static final String INPUT_QUEUE = "inputQueue";
+    private static final String OUTPUT_QUEUE = "outputQueue";
+
+    public static void main(String[] args) throws Exception {
+        ActiveMQBroker.start();
+        JmsMessageProducer producer = new JmsMessageProducer(INPUT_QUEUE, JmsMessageProducer.DestinationType.QUEUE);
+        producer.start();
+
+        JetInstance jet = Jet.newJetInstance();
+
+        Pipeline pipeline = Pipeline.create();
```

Comment:
we normally use `p` for pipelines, and could we follow the structure outlined here:

https://github.com/hazelcast/hazelcast-jet-code-samples/blob/0.6-maintenance/batch/wordcount/src/main/java/WordCount.java